SwiftUI_DZ/SwiftUI_DZ/AutorizeView/AutorizeView.swift

```diff
@@ -0,0 +1,205 @@
+// AutorizeView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+// Экран авторизации
+struct AutorizeView: View {
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let shopName = "ONLINE FURNITURE STORE"
+        static let failureImageName = "questionmark"
+        static let opacity: (Double, Double, Double) = (0.7, 0.8, 0.9)
+        static let gradientViewHeightDouble = 80.0
+        static let imageSizeDouble = 200.0
+        static let stringUpString = "SING UP"
+        static let inviteLabelString = ("Log in", "Sing up")
+        static let loginPlaceholderString = "+0 (000) 000-00-00"
+        static let passwordTextFieldSring = "Password"
+        static let passwordPlaceholderString = "● ● ● ● ● ● ●"
+        static let forgotPasswordString = "Forgot your password?"
+        static let singInString = "Sing in here"
+        static let tags = (2, 4)
+        static let getStartedButtonWidthHeightValue = (270.0, 80.0)
+        static let getStartedCornerRadiusValue = 35.0
+        static let checkVerificationString = "check verification"
+        static let supportString = "Tech support number +7(911)333-33-33"
+        static let okString = "OK"
+        static let spacerHeigthsDouble = (50.0, 60.0, 70.0)
+        static let textFieldWidthDouble = 280.0
+        static let dividerWidthsDouble = (280.0, 300.0)
+        static let fontSizesDouble = (20.0, 25.0)
+        static let returnString = "Return"
+    }
+    
+    // MARK: - Public Properties
+    
+    var body: some View {
+        VStack {
+            topGradientView
+            loginButtonZStackView
+            Spacer().frame(height: Constants.spacerHeigthsDouble.0)
+            textFieldsVStackView
+            Spacer().frame(height: Constants.spacerHeigthsDouble.1)
+            singUpButtonView
+                .padding(.all)
+            forgotPasswordView
+                .padding(.bottom)
+            checkVerificationButtonView
+            Spacer().frame(height: Constants.spacerHeigthsDouble.1)
+        }
+        .toolbar {
+            ToolbarItem(placement: .keyboard) {
+                Button(Constants.returnString) {
+                    isPasswordFocused = false
+                }
+            }
+        }
+        .navigationBarBackButtonHidden(true)
+        .onTapGesture {
+            isLoginFocused = false
+            isPasswordFocused = false
+        }
+    }
+    
+    // MARK: - Private Properties
+    
+    @StateObject private var autorizeViewModel = AutorizeViewModel()
```

Comment:
```suggestion
    @StateObject private var autorizeViewModel = AutorizeViewModel()
    
```
Отступ между разными враперами